### PR TITLE
ETH-tests: state engine

### PR DESCRIPTION
**Pull request by mrLSD**
Related to issue: #170 

Basic source tests for SpuntikVM: [evm-tests](https://github.com/rust-blockchain/evm-tests)

Cover only: state tests

### CODE REVIEW

File: src/tests/eth/istanbul_builtins.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "name": "alt_bn128_pairing",
+    "pricing": {
+      "0": {
+        "price": { "alt_bn128_pairing": { "base": 100000, "pair": 80000 }}
+      },
+      "0": {
+        "info": "EIP 1108 transition",
+        "price": { "alt_bn128_pairing": { "base": 45000, "pair": 34000 }}
+      }
+    }
```
Comment (joshuajbouw) [inline src/tests/eth/istanbul_builtins.json:41]: I think these should be `"0":` and then `"1:"`. This was taken from EVM tests, right? It is a current issue there that hasn't been resolved yet.

**Review by joshuajbouw — CHANGES_REQUESTED** [review]

File: src/tests/eth/istanbul_builtins.json (hunk 1 above)
Comment (joshuajbouw) [reply src/tests/eth/istanbul_builtins.json:41]: See: https://github.com/rust-blockchain/evm-tests/issues/3

Comment (mrLSD) [reply src/tests/eth/istanbul_builtins.json:41]: I don't use that `json file`.
I used aurora precompiles "backend".

Comment (mrLSD) [reply src/tests/eth/istanbul_builtins.json:41]: @jakestutzman also I can't run tests on upstream. I told it before.
So I can't compare. ((

Comment (joshuajbouw) [reply src/tests/eth/istanbul_builtins.json:41]: Ok then just delete the file.

**Review by joshuajbouw — APPROVED** [review]

**Comment by joshuajbouw** [comment]
These unfortunately can not be merged into this library due to conflicting license from sourced code. Must target a new library entirely. `aurora-engine-tests` will be fine.

**Pull request closed without merging**